Description: A patch for output-pstoedit.c before 0.31.1-16
 This patch has been created to represent all changes was made direct
 to the package before implement dpkg-src 3.0(quilt) format, which has 
 a patch system.
 .
 Fix some warnings from compiler and the use char for 32/64 bits.
Author: Edgar Antonio Palma de la Cruz <xbytemx@gmail.com>
Last-Update: 2011-08-07

--- autotrace-0.31.1.orig/output-pstoedit.c
+++ autotrace-0.31.1/output-pstoedit.c
@@ -276,7 +276,7 @@ output_pstoedit_writer (const at_string
 }
 
 at_bool
-output_pstoedit_is_unusable_writer(const at_string name)
+output_pstoedit_is_unusable_writer(const char *name)
 {
   if (0 == strcmp(name, "sam")
       || 0 == strcmp(name, "dbg")
